src/bandersnatch_filter_plugins/allowlist_name.py

```diff
@@ -121,7 +121,7 @@ def _parse_package_lines(package_lines: List[str]) -> Set[Requirement]:
     filtered_requirements: Set[Requirement] = set()
     for package_line in package_lines:
         package_line = package_line.strip()
-        if not package_line or package_line.startswith("#"):
+        if not package_line or package_line.startswith("#") or package_line.startswith("-"):
```

Comment:
```suggestion
        if not package_line or package_line.startswith(("#", "-"))):
```
Pretty sure this works - I only tested in 3.10.6 tho ... Don't change if 3.8 can't handle this.